README.md

```diff
@@ -46,7 +56,7 @@ generate_ethnicity_cohort:
       cohort: output/input_ethnicity.csv
 ```
 
-The following cohort-joiner reusable action joins the cohorts, and then saves them to an output directory.
+The following cohort-joiner reusable action joins the cohorts, and then saves them to a new output directory.
```

Comment:
what do you think about adding "as a single file"? ... and saves them (as a single file) to a new output directory ...

Reply:
I think that could be confusing. cohort-joiner saves each left-hand/right-hand pair as a single file, but there will be multiple single files when there are multiple left-hand files.